src/components/tutorial-card/connected-tutorial-card.tsx

```diff
@@ -4,9 +4,8 @@ import TutorialCard from '.'
 import { TutorialCardPropsWithId } from '.'
 
 export function ConnectedTutorialCard(props: TutorialCardPropsWithId) {
-	/**
-	 * const isBookmarked = useIsBookmarked(props.id)
-	 */
+	const { id: tutorialId } = props
+
 	return (
 		<TutorialCard
 			{...props}
```

Comment:
```suggestion
	const { id: tutorialId, ...rest } = props

	return (
		<TutorialCard
			{...rest}
```
Since we don't need to pass the id to the tutorial card 👍